docs/en/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -79,6 +79,26 @@ The forwarder can ingest logs contained within the payload of an Amazon SQS body
 
 You can set up a separate SQS queue for each type of log. The config parameter for {es} output `es_datastream_name` is mandatory. If this value is set to an {es} data stream, the type of log must be correctly defined with configuration parameters. A single configuration file can have many input sections, pointing to different SQS queues that match specific log types.
 
+[discrete]
+[[at-least-once-delivery]]
+== At-least-once delivery
+
+Elastic Serverless Forwarder ensures "at-least-once" delivery of the forwarded message.
```

Comment:
I think this statement should be in the overview, where the Replay queue and Continuing queue are introduced for the first time.